src/shared/pretty-print.c

```diff
@@ -131,9 +131,9 @@ int terminal_urlify_man(const char *page, const char *section, char **ret) {
         return terminal_urlify(url, text, ret);
 }
 
-static int cat_file(const char *filename, bool newline) {
+static int cat_file(const char *filename, bool newline, CatFlags flags) {
```

Comment:
can you also move `newline` into CatFlags if you touch this? even if callers will not typically set it, and we'd only set it internally i still think it makes sense to expose as flag, simply for uniformity.

Reply:
Eww, right now it is an internal detail that is only exposed in the static function. If it was a flag, it'd need to be moved to the public header. I don't think this is better.